RELEASING.md

```diff
@@ -0,0 +1,84 @@
+# Creating a new release
+
+This document explains how to create a new release. In future
+versions of this packages, some of the steps below will be automatized.
+
+## Preparing your environment for a release
+
+Make sure that your local environment is ready for a full release on
+PyPI and conda. In particular:
+
+- you should install the packages
+[`pypandoc`](https://pypi.python.org/pypi/pypandoc/),
+[`twine`](https://pypi.python.org/pypi/twine),
+[`conda-build`](http://conda.pydata.org/docs/commands/build/conda-build.html)
+and [`anaconda-client`](https://anaconda.org/anaconda/anaconda-client)
+- you should have a registered account on [PyPI](https://pypi.python.org/pypi) and [test PyPI](https://testpypi.python.org/pypi), and your `$HOME` should contain a file `.pypirc` which contains the following text:
+  ```
+[distutils]
+index-servers=
+	pypitest
+	pypi
+
+[pypitest]
+repository = https://testpypi.python.org/pypi
+username = <yourPypiUsername>
+
+[pypi]
+repository = https://pypi.python.org/pypi
+username = <yourPypiUsername>
+```
+- you should have a registered account on [Anaconda.org](https://anaconda.org/)
+
+## Creating a release on Github
+
+- Make sure that the version number in `setup.py`
+  corresponds to the new release, and that the corresponding changes have been
+  documented in `CHANGELOG.md`.
+
+- If everything works fine, then merge the `dev` version into `master`
+and upload it to Github:
+```
+git checkout master
+git merge dev
+git push
+```
+
+- Create a new release through the graphical interface on Github
```

Comment:
I actually just found out that tagging locally is even better, since you can sign your releases with your gpg key. One can then still draft a release with the GitHub API for the `git push --tags` one just uploaded (which is just an alias for the sha again).

We can keep that in mind, but I will personally start to sign all upcoming releases I maintain at HZDR.